cli/templatepush_test.go

```diff
@@ -106,7 +106,7 @@ func TestTemplatePush(t *testing.T) {
 		inv = inv.WithContext(ctx)
 		w := clitest.StartWithWaiter(t, inv)
 
-		pty.ExpectNoMatchBefore(ctx, "Template message is longer than 72 characters", "Updated version at")
+		pty.ExpectNoMatchBefore(ctx, "Template message is longer than 72 characters", "Updated version")
```

Comment:
Can we make sure that the template version matches the one exposed via API, and match the whole phrase `Updated version XYZ at`?

EDIT:
I guess we could match the timestamp too, but it isn't crucial.